src/app/views/sessions/signin/snackbar/insecure-connection.component.html

```diff
@@ -1,8 +1,4 @@
 <ix-icon name="info"></ix-icon>
-<span class="message">{{ 'You are using an insecure connection. Switch to HTTPS for secure access.' | translate }}</span>
-<button mat-button ixTest="configure" (click)="onConfigure()">
-  <span class="text-configure">{{ 'Configure SSL' | translate }}</span>
-</button>
-<button mat-icon-button ixTest="close" (click)="onClose()">
-  <ix-icon name="close"></ix-icon>
-</button>
+<span class="message">
+  {{ 'You are using an insecure connection. Switch to HTTPS for secure access.' | translate }}
```

Comment:
Let's add a link to `Switch to HTTPs` string so it's clickable with the https link redirection

Reply:
> Let's add a link to `Switch to HTTPs` string so it's clickable with the https link redirection

How will user see the FORM if he's not logged in yet❓

Reply:
I don't understand what you mean. I'm saying make it something like
`'You are using an insecure connection. <a href="https://naslink.com">Switch to HTTPS</a> for secure access.'`

Reply:
Oh, I see.
Which link should be there?
Before it worked like - If I click a LINK - sidenav opened with a form, I thought you mean open form again.